audio-worklet/lib/param-controller.js

```diff
@@ -35,7 +35,17 @@ class ParamController {
     this.id_ = options.id || this.name_;
     this.controller_.type = options.type || 'range';
     this.controller_.min = options.min || 0;
-    this.controller_.max = options.max || 0;
+
+    if (options.max == null)
+      this.controller_.max = 1;
+    else
+      this.controller_.max = options.max;
+
+    if (options.step == null)
+      this.controller_.step = 0.1;
+    else
+      this.controller_.step = options.step;
+
     this.controller_.step = options.step || 0;
```

Comment:
You can't have both lines 44-48 and this line 49.  What's wrong with `...step = options.step || 0.1`?